docs/etcd3-migration.md

```diff
@@ -63,7 +80,8 @@ more disruption and are not necessarily lower risk.
 
 ### Adopt etcd-manager with kops 1.11 / kubernetes 1.11
 
-To adopt etcd-manager before kops 1.12 & kubernetes 1.12, you
+If you don't already have TLS enabled with etcd, you can adopt etcd-manager before
```

Comment:
I put this qualifier in here because I received this error message when trying to use etcd-manager with kops 1.11.0 and 1.11.1. This could have been user error 🤷‍♂️ 

```
"error building tasks: TLS not supported for etcd-manager"
```

Reply:
Agreed, we ran into that as well.